cli/connect/console.go

```diff
@@ -247,6 +247,14 @@ func getExecutor(console *Console) prompt.Executor {
 		var data string
 		if _, err := console.conn.Eval(consoleEvalFuncBody, args, opts); err != nil {
 			if err == io.EOF {
+				if strings.Contains(console.input, "os.exit(") {
+					console.Close()
```

Comment:
I haven't dug deep, but the same problem has already been fixed in https://github.com/tarantool/tt/commit/f1111a928dca63641f99db3b358ca91dec2769ac .
Maybe we should
- move the code fixing the console into a separate function
- call it from `console.close()`.

Reply:
https://github.com/tarantool/tt/commit/f1111a928dca63641f99db3b358ca91dec2769ac  Doesn't fix it. Moved code